packages/unified-latex-ctan/package/minted/libs/command-argument-parser.ts

```diff
@@ -0,0 +1,45 @@
+import { arg } from "@unified-latex/unified-latex-builder";
+import { Argument, ArgumentParser } from "@unified-latex/unified-latex-types";
+import { parse as parseArgspec } from "@unified-latex/unified-latex-util-argspec";
+import { gobbleSingleArgument } from "@unified-latex/unified-latex-util-arguments";
+import { match } from "@unified-latex/unified-latex-util-match";
+
+/**
+ * This argument parser parses arguments in the form of
+ * - [⟨options⟩]{⟨language⟩}⟨delim⟩⟨code⟩⟨delim⟩
+ * - [⟨options⟩]{⟨language⟩}{⟨code⟩}
+ */
+export const argumentParser: ArgumentParser = (nodes, startPos) => {
```

Comment:
Can we rename the function or the file so that they both are the same? E.g. `commandArgumentParser` or rename the file to `argument-parser.ts`.